.travis.yml

```diff
@@ -118,8 +118,8 @@ script:
   - if [[ $INSTALL_TYPE == 'conda' ]]; then py.test --durations=20 --cov devito tests/; fi
   # Run remaining specialized examples and tests
   # Additional seismic operator tests
-  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P tti -so 4 -a -d 20 20 20 -n 5; fi
-  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P acoustic -a; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P tti -so 4 -d 20 20 20 -n 5; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P acoustic; fi
```

Comment:
why no more autotuning?

Reply:
oh, it's like before. Autotuning is on by default now

Reply:
ok